### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pkropf**
Added installation note to migrate the database.

### CODE REVIEW

**Comment by jeffgdotorg** [comment]
This PR addresses issue #18

File: README.md [hunk 1]
```diff
@@ -34,6 +34,11 @@ Enable the plugin in /opt/netbox/netbox/netbox/configuration.py:
 ```
 PLUGINS = ['netbox_floorplan']
 ```
+Update the database:
```
Comment (jeffgdotorg) [inline README.md:37]:
```suggestion
To add the required `netbox_floorplan` tables to your NetBox database, run the `migrate` manager subcommand in the NetBox virtual environment:
```

File: README.md [hunk 2]
```diff
@@ -34,6 +34,11 @@ Enable the plugin in /opt/netbox/netbox/netbox/configuration.py:
...
+```
+python manage.py migrate netbox_floorplan
```
Comment (jeffgdotorg) [inline README.md:39]:
```suggestion
cd /opt/netbox
sudo ./venv/bin/python3 netbox/manage.py migrate
```

**Review by jeffgdotorg — CHANGES_REQUESTED** [review]
Thanks for creating this PR. I'll assign #18 to you and merge once requested changes are made.

**Comment by cruse1977** [comment]
Closing this as superceded.

**Pull request closed without merging**